CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### Tools helper code
 
-* Updated Blacklist of synced pipelines
```

Comment:
Removing stuff from the CHANGELOG?

Reply:
Yes, it doesn’t make sense to say in the changelog for this release that we have updated the blacklist, when a couple of lines later say that it’s removed..